frontend/utils/errorHandlers.js

```diff
@@ -0,0 +1,3 @@
+export function logExceptNavigationDuplicated (err: Object) {
+  err.name !== 'NavigationDuplicated' && console.error(err)
+}
```

Comment:
This function [used to exist in `frontend/controller/utils/misc.js`](https://github.com/okTurtles/group-income-simple/pull/690/files#diff-b13300ef486717f682409d5ff465964df6f67442ecb77abc6a51da8e8256ecd3R10).

I would put it back there instead of creating a new file just for it.

Reply:
Sure. I will